scripts/ci/github_actions/driver/setup_ci_dependencies.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -xe
+set +o pipefail
+
+install_ci_dependencies (){
+  python -m pip install --upgrade pip
+  echo docker-hub==2.2.0 > dev-requirements.txt
+  pip install -r dev-requirements.txt
+}
+
+get_driver_version (){
+  source /home/runner/.bash_profile
+  cd common
+  image_version=`yq eval .identity.version config.yaml`
+  cd -
```

Comment:
wouldn't `yq eval .identity.version common/config.yaml` be enough?

Reply:
no, since using yq as a docker image, you need to get into the directory of the file

Reply:
are you sure? it works for me

Reply:
my bad, done